cax-ap-cam/LICENSE

```diff
@@ -0,0 +1,674 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
this file should be a link to the global LICENSE file

Reply:
done